razzle.config.js

```diff
@@ -8,15 +11,21 @@ module.exports = {
         A prod build will fail if the API changes so it is fairly safe.
       */
       appConfig.module.rules.shift();
-      // This is to pass the bundlde performance test in CI
-      appConfig.performance = Object.assign(
-        {},
-        {
-          maxAssetSize: 350000,
-          maxEntrypointSize: 350000,
-        },
+    } else if (target === 'node' && dev) {
```

Comment:
If you remove this requirement and update the generator to save to static files instead of starting a server, then the static html and json files should be generated on each `npm run build`.